phplib/admin-ratty.php

```diff
@@ -230,7 +230,7 @@ function display($self_link) {
                     list($field_name, $field_description, $field_examples) = $row;
                     print "<b>$field_name:</b> $field_description. ";
                     print "e.g. " . implode(", ", array_map(
-                    create_function('$a', 'return "\'".trim_characters($a, 0, 50)."\'";'), $field_examples));
+                    function($a) { return "\'".trim_characters($a, 0, 50)."\'"; }, $field_examples));
```

Comment:
You need to not escape the single quotes any more.

Reply:
I've pushed a fixup commit, assuming that's all good I'll squash it in before merging.